server/views/pages/bookAVisit/overbooking.njk

```diff
@@ -14,36 +15,34 @@
       <p>
         <span data-test="bookings-count">{{ bookingsCount }}</span> of
         <span data-test="max-capacity">{{ maxCapacity }}</span> {{ "table" | pluralise(maxCapacity) }} are booked for
-        <span data-test="visit-start-time">{{ visitStartTime }}</span> to
-        <span data-test="visit-end-time">{{ visitEndTime }}</span> on 
-        <span data-test="visit-date">{{ visitStartTime | formatDate("EEEE d MMMM") }}</span>.
+        <span data-test="visit-start-time">{{ visitSession.startTimestamp | formatTime }}</span> to
+        <span data-test="visit-end-time">{{ visitSession.endTimestamp | formatTime }}</span> on 
+        <span data-test="visit-date">{{ visitSession.startTimestamp | formatDate("EEEE d MMMM") }}</span>.
       </p>
+      <form action="{{ urlPrefix }}/confirm-overbooking" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+        {{ govukRadios({
+          name: "confirmOverbooking",
+          items: [
+            {
+              value: "yes",
+              text: "Yes, book for this timeslot"
+            },
+            {
+              value: "no",
+              text: "No, choose a new date and time"
+            }
+          ],
+          errorMessage: errors | findError('confirmOverbooking'),
+          attributes: { "data-test": "overbooking-confirm" }
```

Comment:
```suggestion
          attributes: { "data-test": "confirm-overbooking" }
```
Very minor point - words here the other way round to elsewhere in the code. Doesn't matter which way - just consistency would be better!